examples/isomorphic-flux-chat/js/main.js

```diff
@@ -0,0 +1,25 @@
+var mockData = require('./mock-data')
+var Nuclear = require('nuclear-js')
+var reactor = new Nuclear.Reactor({
+  debug: true,
+})
+window.reactor = reactor
+var Chat = require('./modules/chat')
+
+var ChatApp = require('./components/ChatApp.react')
+ChatApp = Nuclear.provideReactor(ChatApp)
```

Comment:
I was looking for something better but ... nothing seemed better than just passing it as a first argument. That way it seemd clear to me that actions are actions on THAT reactor. Instead of before actions on the singleton =).

https://github.com/jordangarcia/nuclear-js-react-addons/pull/2 btw.

And please comment on https://github.com/optimizely/nuclear-js/pull/129 (opened) instead of this one (closed) :stuck_out_tongue: